### PR TITLE
feature: load list food

### PR DESCRIPTION
**Pull request by at-sonvu**

### CODE REVIEW

**Comment by at-daiho** [comment]
:+1: There are no linter violations.

File: MyApp/View/Cell/SuggestionCell.swift [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+//
+//  Created by Son Vu L.L on 7/27/17.
+//  Copyright © 2017 Asian Tech Co., Ltd. All rights reserved.
+//
+
+import UIKit
+
+class SuggestionCell: UITableViewCell {
+
+    @IBOutlet weak var itemImageView: UIImageView!
```
Comment (at-tuanmai2) [inline MyApp/View/Cell/SuggestionCell.swift]: change name

File: MyApp/View/Controllers/Home/HomeViewController.swift [hunk 2]
```diff
@@ -78,8 +78,20 @@ final class HomeViewController: UIViewController, MVVM.View {
...
+
+    @IBAction func loadSuggestDrinksButtonTouchUpInside(_ sender: UIButton) {
+        let vc = ListMenuViewController()
+        vc.viewModel = viewModel.listViewModelWith(type: .Drink)
+        self.navigationController?.pushViewController(vc, animated: true)
+    }
+
     @IBAction func loadMoreFoodButtonTouchUpInside(_ sender: UIButton) {
-        viewModel.loadMoreFoodButtonTouchUpInside(vc: self)
+        self.navigationController?.pushViewController(SuggestionViewController(), animated: true)
```
Comment (at-tuanmai2) [inline MyApp/View/Controllers/Home/HomeViewController.swift]: viết vc riêng ra (let vc = Su..())

File: MyApp/View/Controllers/Suggestion/SuggestionViewController.swift [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+    @IBOutlet weak var leadingConstraint: NSLayoutConstraint!
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        configTableView()
+    }
+    @IBAction func backNavigationButtonTouchUpInside(_ sender: UIButton) {
+        _ = navigationController?.popViewController(animated: true)
+    }
+    @IBAction func foodButtonTouchUpInside(_ sender: UIButton) {
+        self.leadingConstraint.constant = 0
```
Comment (at-tuanmai2) [inline MyApp/View/Controllers/Suggestion/SuggestionViewController.swift]: remove self

File: MyApp/View/Controllers/Suggestion/SuggestionViewController.swift [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+        UIView.animate(withDuration: 1.0, delay: 0.0, usingSpringWithDamping: 0.4, initialSpringVelocity: 10.0, options: .curveEaseIn, animations: {
+            self.view.layoutIfNeeded()
+        })
+
+    }
+    @IBAction func drinkButtonTouchUpInside(_ sender: UIButton) {
+        self.leadingConstraint.constant = UIScreen.main.bounds.width / 2
```
Comment (at-tuanmai2) [inline MyApp/View/Controllers/Suggestion/SuggestionViewController.swift]: xoa self

File: MyApp/View/Controllers/Suggestion/SuggestionViewController.swift [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+        UIView.animate(withDuration: 1.0, delay: 0.0, usingSpringWithDamping: 0.4, initialSpringVelocity: 10.0, options: .curveEaseIn, animations: {
+            self.view.layoutIfNeeded()
+        })
+    }
+
+    func configTableView() {
+        tableView.dataSource = self
+        tableView.delegate = self
+        tableView.register(SuggestionCell.self)
```
Comment (at-tuanmai2) [inline MyApp/View/Controllers/Suggestion/SuggestionViewController.swift]: register lên trước .datasource

**Pull request merged**